charts/backstage/values.yaml

```diff
@@ -108,8 +108,8 @@ upstream:
       - name: POSTGRESQL_ADMIN_PASSWORD
         valueFrom:
           secretKeyRef:
-            key: postgres-password
-            name: '{{- include "janus-idp.postgresql.secretName" . }}'
+            key: '{{- include "postgresql.adminPasswordKey" . }}'
+            name: '{{- include "postgresql.v1.secretName" . }}'
```

Comment:
using this helper here changes the `name` property from `rhdh-chart-backstage` to `rhdh-chart-postgresql`. Is this a breaking change / problem?